modules/layers/src/text-layer/multi-icon-layer/multi-icon-layer.js

```diff
@@ -80,25 +81,37 @@ export default class MultiIconLayer extends IconLayer {
 
   updateState(updateParams) {
     super.updateState(updateParams);
-    const {changeFlags} = updateParams;
+    const {changeFlags, oldProps, props} = updateParams;
 
     if (
       changeFlags.updateTriggersChanged &&
       (changeFlags.updateTriggersChanged.getAnchorX || changeFlags.updateTriggersChanged.getAnchorY)
     ) {
       this.getAttributeManager().invalidate('instanceOffsets');
     }
+
+    if (props.backgroundColor !== oldProps.backgroundColor) {
+      const backgroundColor = props.backgroundColor.map(c => c / 255.0);
+      if (backgroundColor.length === 3) {
```

Comment:
Alpha is no longer used, make `backgroundColor` a vec3, and `props.backgroundColor` default to `null`.

Reply:
`Uncaught Error: Invalid prop backgroundColor: null`. Why? the property is optional.